apps/aecore/lib/aecore/utils/serialization.ex

```diff
@@ -39,4 +39,20 @@ defmodule Aecore.Utils.Serialization do
         Base.decode16!(data)
     end
   end
+
+  @spec txs(map(), :serialize | :deserialize) :: map() | {:error, term()}
+  def txs(tx, direction) do
+    case direction do
+      :serialize -> 
+        tx 
+        |> :erlang.term_to_binary()
```

Comment:
like this all of the transaction will be a base64 string, only the binary fields should be handled like this, so a readable json format is used. See the Serialize.block() function.